app/scripts/modules/core/src/cluster/allClusters.controller.js

```diff
@@ -123,6 +123,7 @@ module(CORE_CLUSTER_ALLCLUSTERS_CONTROLLER, [
               $uibModal.open({
                 templateUrl: providerConfig.cloneServerGroupTemplateUrl,
                 controller: `${providerConfig.cloneServerGroupController} as ctrl`,
+                windowClass: 'modal-z-index-fix',
```

Comment:
```suggestion
                windowClass: 'layer-critical',
```
```suggestion
                windowClass: 'modal-z-index',
```

Reply:
From Angular's bootstrapping directives, windowClass (Type: string) - Additional CSS class(es) to be added to a modal window template. 

Based on the above description, it seems like it should be supplemental, but it doesn't work when I switch over to using the layout-critical class. I've updated the PR to rename the class to modal-z-index.

Reply:
```suggestion
                windowClass: 'modal-z-index',
```
This should now just be `modal-z-index`, right? Otherwise LGTM, thanks for the quick fix!

Reply:
Good catch! Not sure how this file didn't commit with the update everywhere else, but everything should be updated now!

Reply:
Great, thanks! Going to merge this now.